src/shogun/mathematics/Statistics.cpp

```diff
@@ -573,9 +574,7 @@ float64_t CStatistics::erfc8_weighted_sum(float64_t x)
 
 float64_t CStatistics::normal_cdf(float64_t x, float64_t std_dev)
 {
-	SG_SERROR("NOT IMPLEMENTED");
-	return 0;
-//	return 0.5*(error_function_complement(-x/std_dev/1.41421356237309514547));
```

Comment:
@yorkerlin I think since this is in the standard, we're good to go. Have a look at the discussion [here](http://stackoverflow.com/questions/631629/erfx-and-math-h) and the implementation [here](http://www.netlib.org/fdlibm/s_erf.c). Looks like they take quite a lot of measures for different cases. I'll add the unit tests though.

One question : are these numbers (for the `lnormal_cdf` case)
```c++
t[0]=0.197197002817524946749;
t[1]=0.0056939283141627128337;
t[2]=11.17348207854067787537;
t[3]=0.513878566283254256675;
t[4]=4.34696415708135575073;
t[5]=-8.69392831416271150147;
t[6]=-17.38785662832542300293;
t[7]=-34.42874909956949380785;
t[8]=-67.46964157081356461276;
t[9]=-136.67410392703391153191;
```
just chosen randomly while writing the unit-test or there is some source that I should be looking at?